pyproject.toml

```diff
@@ -1,15 +1,15 @@
 [build-system]
 requires = [
-    "setuptools>=64",
-    "wheel",
+    "scikit-build-core>=0.10,<=0.10.3",
     # On Windows, use the CasADi vcpkg registry and CMake bundled from MSVC
-    "casadi>=3.6.5; platform_system!='Windows'",
+    "casadi>=3.6.5,<=3.6.6; platform_system!='Windows'",
```

Comment:
It was decided that it would be better to pin all build-time dependencies earlier. But we don't need to do that. Just pinning scikit-build-core is good. 

Fixing this!